packages/plugins/rubygem-logify/rubygem-logify.spec

```
@@ -1,43 +1,66 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name logify
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.2.0
-Release: 1%{?dist}
-Summary: Logify is an incredibly light-weight Ruby logger
+Release: 2%{?dist}
+Summary: Logify is a lightweight logging library for Ruby!
 Group: Development/Languages
-License: ASL 2.0
+License: Apache 2.0
```

RPM license codes are different from SPDX:

```suggestion
License: ASL 2.0
```